gym_electric_motor/reward_functions/constraint_monitor.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
+import gym
+from ..core import RewardFunction
+from gym.spaces import Box
+
+# todo docstrings, comments for discount factor
+
+
+class ConstraintMonitor:
+    """
+    The ConstraintMonitor Class monitors the system-states and assesses whether
+    they comply the given limits or violate them.
+    It returns the necessary information for the RewardFunction, to calculate
+    the corresponding reward-value.
+    Limits, here called constraints, can be given by the physical limits from
```

Comment:
The idea behind constraints is that they more or less generalize limits as already mentioned in #34 .